Python/pywarpx/WarpXPIC.py

```diff
@@ -52,5 +52,5 @@ def push_Bfields(self, dt):
 
     def apply_particle_boundary_conditions(self):
         libwarpx.mypc_Redistribute() # Redistribute particles
-        libwarpx.warpx_MoveWindow() # !!! not the correct place yet
+        libwarpx.warpx_MoveWindow(self.istep,false) # !!! not the correct place yet
```

Comment:
Ok, but should we pass `move_j` here already? @dpgrote mentioned this but I cannot see it defined in the Python code yet.
```suggestion
        libwarpx.warpx_MoveWindow(self.istep, move_j) # !!! not the correct place yet
```

Reply:
@dpgrote Yes i'm not sure what to put here

Reply:
I would put `true` for now, though it may not matter now since as far as I know, this code is not being used. Putting `true` is safer, to always move J, since it shouldn't cause any problems and is only a possible inefficiency.